src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --text-color: #00acdc;
+  --link-heigh: 60px;
+  width: 100%;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
it's better to set `box-sizing: border-box;` for all elements (use `*` selector)